src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -74,6 +45,17 @@ The following options can be configured as options to the integration, in `new R
 | inlineStylesheet | boolean | `true`  | Should inline stylesheets used in the recording |
 | recordCanvas     | boolean | `false` | Should record `<canvas>` elements |
 
+## Identifying Users
+
+You can use the Sentry SDK to set the user of a session, in order to associate a user identity to a session replay, refer to the <PlatformLink to="/enriching-events/identify-user">Identifying User Docs</PlatformLink>.
+
+Here's a short example using e-mail:
+
+```javascript
+import * as Sentry from "@sentry/browser";
+Sentry.setUser({ email: "sentaur@sentry.dev" });
```

Comment:
```suggestion
Sentry.setUser({ email: "sentaur@sentry.dev" });
```

To spare us another place where we should add platform-specific imports (see my [comment](https://github.com/getsentry/sentry-docs/pull/5899#discussion_r1041993025)), I'd vote to just remove the import here. WDYT?

Reply:
This is a challenge indeed. We went with "always add imports to snippets" a while back because it happens that users come back with "I get an error of Sentry is undefined" or the equivalent in other languages. But the added overhead of having to maintain multiple snippets is indeed an issue.

Your code comment (in the link you shared) can be a good approach. We can also add a couple of examples like:

Reply:
Sure, def not a blocker, merge away 👍 
And I agree, we should discuss this in one place and not multiple 😅